python/CMakeLists.txt

```diff
@@ -40,18 +41,18 @@ if (PYTHON_EXECUTABLE)
     set(__configs ${CMAKE_CONFIGURATION_TYPES})
   endif()
   foreach (__config ${__configs})
-    set(CONFIG ${__config})
-    if(${__config} STREQUAL "Debug")
+    set(CONFIG ${__configs})
+    if(${__configs} STREQUAL "Debug")
       set(PYTHON_PACKAGE_NAME "_phylanxd")
     else()
       set(PYTHON_PACKAGE_NAME "_phylanx")
     endif()
-    configure_file(${SETUP_PY_IN} "${SETUP_PY}_${__config}.py" @ONLY)
+    configure_file(${SETUP_PY_IN} "${SETUP_PY}_${__configs}.py" @ONLY)
   endforeach()
 
   configure_file(${INIT_PY_IN} "${INIT_PY}" @ONLY)
 
-  set(__setup_script "${SETUP_PY}_$<CONFIG>.py")
+  set(__setup_script "${SETUP_PY}_${__configs}.py")
```

Comment:
Those changes `__config` --> `__configs` are definitely not correct. On multi-target platforms (i.e. XCode or Visual Stdio you have more than one possible configuration, for instance `${__configs} == "Debug;Release"`

Reply:
I see.
Well, __config was not getting set.

Reply:
Ah, it just needs to be in a loop

Reply:
It was getting set [here](https://github.com/STEllAR-GROUP/phylanx/pull/36/files/f34b9d6a1aaf4f8052f144b7efde431b92a06a6c#diff-79b695dff65b8b0a69bfed14e824cb18R43)

Reply:
Is `CMAKE_BUILD_TYPE` set for you?

Reply:
Yes, I set CMAKE_BUILD_TYPE explicitly, give me a sec....

Reply:
OK, see if this new one works better for you.